src/Interfaces/Accessibility/AccessibilityInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+declare(strict_types=1);
+
+namespace CodeFoundation\FlowConfig\Interfaces\Accessibility;
+
+interface AccessibilityInterface
```

Comment:
Love the concept, not the name. `Accessibility` has strong connotations of HCI. ie; can blind / deaf / etc users use this, which I don't think applies in this instance.

Reply:
fair call, any thoughts on a better name for this interface?

Reply:
`AccessControlInterface`, or `AuthorizationInterface` ? Second one sounds like roles and permissions for roles/permissions access of users, which is not entirely related.

Reply:
how about `KeyAccessInterface`?